Mjml.Net/Mjml.Net.csproj

```diff
@@ -13,11 +13,11 @@
 
   <ItemGroup>
     <PackageReference Include="HtmlPerformanceKit" Version="1.0.0" />
-    <PackageReference Include="Meziantou.Analyzer" Version="2.0.145">
+    <PackageReference Include="Meziantou.Analyzer" Version="2.0.184">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="8.0.2" />
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="9.0.0" />
```

Comment:
Can we have a multi target here?

Reply:
I think the change should work now, although not sure if you want to target v8 for anything less than .net 9